organization/_posts/2020/2020-05-07-coordination.md

```diff
@@ -0,0 +1,162 @@
+
+### Detector Simulation
+
+- We are targeting a working group meeting on 27th May or 3rd June. Currently discussing which topic we want to focus on. In addition to the list of topics we had penciled in we are considering those that were to be addressed in the original Lund workshop.
+
+
+### Reconstruction and Software Triggers
+
+- Meetings for the foreseeable future: connect to LHCC document topics
+    - June: GPU in trigger
+    - July or August: Tracking algorithms exploiting timing information 
+[Note: take one of the two months as “vacation”] 
+    - September: reconstruction algorithms in experiments connected to LHC
+    - October: ML inference in trigger 
+    - November: particle flow commissioning up to Run-3 
+
+- Common trigger&reco page updates:
+    - As discussed, would like to add links and descriptions for projects that have been added to the LHCC document [[link to text](https://docs.google.com/document/d/10D5Lf7NWfd5FvS9OJzcNPw_t1xbEn5gXRZoDCTk8QEs/edit#heading=h.penorw1ehh1k)]
+    - How to distinguish what is endorsed and hosted by HSF, if any? See discussion above
+
+### PyHEP
+- We want to finalise the preparations for the workshop but things have been moving more slowly than usual, understandably.
+- Our community got 3 talks and one poster at SciPy 2020, and this strong presence is excellent news.
+    - Standard registration for the whole conference is now just $75 ($25 for students)! Do consider attending.
+
+### Software Tools and Packaging
+
+* Ben Morgan gave a nice presentation on Spack at the pre-GDB meeting on May 5th ([slides](https://indico.cern.ch/event/813800/contributions/3830178/attachments/2031805/3400697/SpackGDBMeeting_1.pdf)).
+    * Perhaps it'll be beneficial to have a meeting to discuss the latest developments on the packaging/Spack side. In particular, it would be nice to review the recent developments, survey the missing packages that we want to see in the Spack repository etc. This was also brought up [previously](https://hepsoftwarefoundation.org/organization/2020/03/26/coordination.html) and the suggestion was to collect LCG release contents as a start.
+    * News from [Key4hep project](https://indico.cern.ch/event/912631/): Valentin now has the whole Key4hep stack building in Spack (not bootstrapping from LCG release).
+* On the tools side we're going to be adding an optional GPU monitoring support to `prmon` (see issue [#105](https://github.com/HSF/prmon/issues/105)). The main idea is to initially support NVidia and use `nvidia-smi` for collection. Then extend it to support other hardware as well.
+
+
+### Software Training
+
+- Will organise virtual training soon. Recorded talks, with small group work a few days after.
+
+### Event generators
+* Updating the generator paper. Will submit to the journal next week (still waiting for some comments).
+    * Also plan to contribute it to the Snowmass process.
```

Comment:
```suggestion
    * Also plan to contribute to the Snowmass process.
```

Reply:
"it" = "the generator paper", which I think is ok